packages/moonstone/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/Scroller` to support RTL to prevent scrolls originating from focus events
```

Comment:
What about this?
- `moonstone/Scroller` to prevent scrolls by focus properly regarding RTL languages.